packages/components/src/components/hds/app-header/index.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
+import { registerDestructor } from '@ember/destroyable';
+
+export interface HdsAppHeaderSignature {
+  Blocks: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    logo?: any;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    globalActions?: any;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Question: sorry, not a11y related - but why are these typed as any?

Reply:
Because we aren't restricting what content can go in them.

Reply:
One is typed as an array the other isn't. Do you have a suggestion about what it should be?

Anyway I'm not a Typescript expert. We only started using TypeScript recently on the team and I'd only been using it for a short time before joining...

Reply:
Hahah I think this is where me being new to ember is confusing me.. so feel free to ignore! 

I just noticed in some components where it seems like we allow any content to be passed, it used the array syntax (example: https://github.com/hashicorp/design-system/blob/e4cf01addb3c650a2e1871e575db2d7bf27e7b3e/packages/components/src/components/hds/page-header/title.ts#L12)

Reply:
I mean, an empty array could very well be better to use.

Searching our code now, I'm seeing several examples of that so I'll see if I can change it and then remove the eslint disable comment which would be good.

Reply:
That ended up working, thanks! (Sorry I just didn't understand the context of your original question since I'm new to TypeScript.)

Reply:
You're good! I should have added more context.